tests/EvmAcceptanceTests/contracts/BasicInterop.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.7.0 <0.9.0;
+
+pragma abicoder v2;
+
+contract BasicInterop {
+
+    function callSimpleMap(address contract_address, string memory tran_name, address recipient, uint128 amount) public {
+        bytes memory encodedArgs = abi.encode(contract_address, tran_name, recipient, amount);
+        uint256 argsLength = encodedArgs.length;
+        bool success;
+        assembly {
+            success := call(21000, 0x98, 0, add(encodedArgs, 0x20), argsLength, 0x20, 0)
```

Comment:
Maybe some comments about what this is doing, so we can check that it is doing what you think it's doing?
(also because this test is the only docs, and it's a bit obscure how to use the interop from the test!)